<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/icons.qrc" >
    <normaloff>:/QBoard/icon/qub.png</normaloff>:/QBoard/icon/qub.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>60</y>
     <width>675</width>
     <height>402</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" >
    <item row="1" column="0" >
     <widget class="QWidget" native="1" name="clientArea" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>675</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionHome" />
    <addaction name="actionNewWindow" />
    <addaction name="separator" />
    <addaction name="actionSave" />
    <addaction name="actionLoad" />
    <addaction name="actionPrint" />
    <addaction name="separator" />
    <addaction name="actionToggleBrowserView" />
    <addaction name="actionRefreshFileList" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="title" >
     <string>Edit</string>
    </property>
    <addaction name="actionSelectAll" />
    <addaction name="separator" />
    <addaction name="actionCopy" />
    <addaction name="actionCut" />
    <addaction name="actionPaste" />
    <addaction name="actionClearClipboard" />
   </widget>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp" />
    <addaction name="actionAboutQBoard" />
    <addaction name="actionAboutQt" />
   </widget>
   <widget class="QMenu" name="menu_Board" >
    <property name="title" >
     <string>&amp;Board</string>
    </property>
    <addaction name="actionToggleBoardDragMode" />
    <addaction name="actionNewBoardView" />
    <addaction name="separator" />
    <addaction name="actionZoomIn" />
    <addaction name="actionZoomOut" />
    <addaction name="actionZoomReset" />
    <addaction name="actionRotate90" />
    <addaction name="separator" />
    <addaction name="actionClearBoard" />
   </widget>
   <widget class="QMenu" name="menuPieces" >
    <property name="title" >
     <string>&amp;Pieces</string>
    </property>
    <addaction name="actionAddQGIHtml" />
    <addaction name="actionAddLine" />
    <addaction name="separator" />
    <addaction name="actionClearBoard" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menu_Board" />
   <addaction name="menuPieces" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>462</y>
     <width>675</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>26</y>
     <width>675</width>
     <height>34</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionToggleBrowserView" />
   <addaction name="actionRefreshFileList" />
   <addaction name="actionHome" />
   <addaction name="separator" />
   <addaction name="actionLoad" />
   <addaction name="actionSave" />
   <addaction name="actionPrint" />
   <addaction name="separator" />
   <addaction name="actionToggleBoardDragMode" />
   <addaction name="actionAddQGIHtml" />
   <addaction name="actionAddLine" />
   <addaction name="actionNewBoardView" />
   <addaction name="actionZoomOut" />
   <addaction name="actionZoomIn" />
   <addaction name="actionZoomReset" />
   <addaction name="actionRotate90" />
   <addaction name="actionClearBoard" />
   <addaction name="separator" />
   <addaction name="actionExperiment" />
  </widget>
  <action name="actionSave" >
   <property name="icon" >
    <iconset>
     <normalon>:/QBoard/icon/filesaveas.png</normalon>
    </iconset>
   </property>
   <property name="text" >
    <string>&amp;Save...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad" >
   <property name="icon" >
    <iconset>
     <normalon>:/QBoard/icon/fileopen.png</normalon>
    </iconset>
   </property>
   <property name="text" >
    <string>&amp;Load...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset>
     <normalon>:/QBoard/icon/stop.png</normalon>
    </iconset>
   </property>
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCopy" >
   <property name="icon" >
    <iconset>
     <normalon>:/QBoard/icon/editcopy.png</normalon>
    </iconset>
   </property>
   <property name="text" >
    <string>Copy</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut" >
   <property name="icon" >
    <iconset>
     <normalon>:/QBoard/icon/editcut.png</normalon>
    </iconset>
   </property>
   <property name="text" >
    <string>Cut</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>
     <normalon>:/QBoard/icon/editpaste.png</normalon>
    </iconset>
   </property>
   <property name="text" >
    <string>Paste</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionClearClipboard" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>
     <normalon>:/QBoard/icon/editshred.png</normalon>
    </iconset>
   </property>
   <property name="text" >
    <string>Clear clipboard</string>
   </property>
  </action>
  <action name="actionZoomIn" >
   <property name="icon" >
    <iconset>
     <normalon>:/QBoard/icon/viewmag+.png</normalon>
    </iconset>
   </property>
   <property name="text" >
    <string>Zoom &amp;in</string>
   </property>
   <property name="shortcut" >
    <string>+</string>
   </property>
  </action>
  <action name="actionZoomOut" >
   <property name="icon" >
    <iconset>
     <normalon>:/QBoard/icon/viewmag-.png</normalon>
    </iconset>
   </property>
   <property name="text" >
    <string>Zoom &amp;out</string>
   </property>
   <property name="shortcut" >
    <string>-</string>
   </property>
  </action>
  <action name="actionZoomReset" >
   <property name="icon" >
    <iconset>
     <normalon>:/QBoard/icon/viewmag1.png</normalon>
    </iconset>
   </property>
   <property name="text" >
    <string>Zoom &amp;100%</string>
   </property>
   <property name="shortcut" >
    <string>1</string>
   </property>
  </action>
  <action name="actionRefreshFileList" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset>
     <normalon>:/QBoard/icon/reload.png</normalon>
    </iconset>
   </property>
   <property name="text" >
    <string>&amp;Refresh file view</string>
   </property>
   <property name="iconText" >
    <string>Refresh file view</string>
   </property>
   <property name="toolTip" >
    <string>Refresh file view</string>
   </property>
   <property name="shortcut" >
    <string>F5</string>
   </property>
  </action>
  <action name="actionToggleBrowserView" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources/icons.qrc" >
     <normaloff>:/QBoard/icon/view_sidetree.png</normaloff>
     <normalon>:/QBoard/icon/view_remove.png</normalon>:/QBoard/icon/view_sidetree.png</iconset>
   </property>
   <property name="text" >
    <string>Toggle file &amp;view</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionNewBoardView" >
   <property name="icon" >
    <iconset>
     <normalon>:/QBoard/icon/window_new.png</normalon>
    </iconset>
   </property>
   <property name="text" >
    <string>New &amp;board view</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExperiment" >
   <property name="icon" >
    <iconset>
     <normalon>:/QBoard/icon/connect_creating.png</normalon>
    </iconset>
   </property>
   <property name="text" >
    <string>Experiment</string>
   </property>
   <property name="shortcut" >
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionHome" >
   <property name="icon" >
    <iconset>
     <normalon>:/QBoard/icon/folder_home.png</normalon>
    </iconset>
   </property>
   <property name="text" >
    <string>&amp;Home</string>
   </property>
   <property name="shortcut" >
    <string>Alt+Home</string>
   </property>
  </action>
  <action name="actionAboutQt" >
   <property name="icon" >
    <iconset>
     <normalon>:/QBoard/icon/qtlogo.png</normalon>
    </iconset>
   </property>
   <property name="text" >
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionAboutQBoard" >
   <property name="icon" >
    <iconset>
     <normalon>:/QBoard/icon/qub-mini.png</normalon>
    </iconset>
   </property>
   <property name="text" >
    <string>&amp;About QBoard</string>
   </property>
  </action>
  <action name="actionPrint" >
   <property name="icon" >
    <iconset>
     <normalon>:/QBoard/icon/fileprint.png</normalon>
    </iconset>
   </property>
   <property name="text" >
    <string>&amp;Print...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionHelp" >
   <property name="icon" >
    <iconset resource="../resources/icons.qrc" >
     <normaloff>:/QBoard/icon/contents.png</normaloff>:/QBoard/icon/contents.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Help</string>
   </property>
   <property name="shortcut" >
    <string>F1</string>
   </property>
  </action>
  <action name="actionClearBoard" >
   <property name="icon" >
    <iconset resource="../resources/icons.qrc" >
     <normaloff>:/QBoard/icon/eraser.png</normaloff>:/QBoard/icon/eraser.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Clear board</string>
   </property>
  </action>
  <action name="actionNewWindow" >
   <property name="icon" >
    <iconset resource="../resources/icons.qrc" >
     <normaloff>:/QBoard/icon/qub-mini.png</normaloff>:/QBoard/icon/qub-mini.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;New Window</string>
   </property>
  </action>
  <action name="actionToggleBoardDragMode" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources/icons.qrc" >
     <normaloff>:/QBoard/icon/mouse.png</normaloff>
     <normalon>:/QBoard/icon/move.png</normalon>:/QBoard/icon/mouse.png</iconset>
   </property>
   <property name="text" >
    <string>Toggle board drag mode</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionRotate90" >
   <property name="icon" >
    <iconset resource="../resources/icons.qrc" >
     <normaloff>:/QBoard/icon/rotate_cw.png</normaloff>:/QBoard/icon/rotate_cw.png</iconset>
   </property>
   <property name="text" >
    <string>Rotate &amp;90 degrees</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAddQGIHtml" >
   <property name="icon" >
    <iconset resource="../resources/icons.qrc" >
     <normaloff>:/QBoard/icon/editclear.png</normaloff>:/QBoard/icon/editclear.png</iconset>
   </property>
   <property name="text" >
    <string>Add &amp;text item</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionAddLine" >
   <property name="icon" >
    <iconset resource="../resources/icons.qrc" >
     <normaloff>:/QBoard/icon/color_line.png</normaloff>:/QBoard/icon/color_line.png</iconset>
   </property>
   <property name="text" >
    <string>Add &amp;Line item</string>
   </property>
  </action>
  <action name="actionSelectAll" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Select &amp;all</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionNewWindow</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>launchNewWindow()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>337</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRotate90</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>rotate90()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>337</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>337</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAddQGIHtml</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>addQGIHtml()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>337</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAddLine</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>addLine()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>337</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>aboutQBoard()</slot>
  <slot>launchHelp()</slot>
  <slot>launchNewWindow()</slot>
  <slot>toggleBoardDragMode(bool)</slot>
  <slot>rotate90()</slot>
  <slot>addQGIHtml()</slot>
  <slot>addLine()</slot>
 </slots>
</ui>
